src/nvim/window.c

```diff
@@ -4918,12 +4917,18 @@ file_name_in_line (
    * Also allow "://" when ':' is not in 'isfname'.
    */
   len = 0;
-  while (vim_isfilec(ptr[len])
-         || ((options & FNAME_HYP) && path_is_url((char *)ptr + len)))
-    if (has_mbyte)
+  while (vim_isfilec(ptr[len]) || (ptr[len] == '\\' && ptr[len + 1] == ' ')
+         || ((options & FNAME_HYP) && path_is_url((char *)ptr + len))) {
+    if (ptr[len] == '\\') {
```

Comment:
It seems that `vim_isfilec('\\')` can be true if `\` is in `isfname` (e.g. on Windows). I've added the check as suggested.